datasets/hda_nli_hindi/hda_nli_hindi.py

```diff
@@ -0,0 +1,154 @@
+}
+"""
+
+_DESCRIPTION = """\
+This dataset is a recasted version of the Hindi Discourse Analysis Dataset used to train models for Natural Language Inference Tasks in Low-Resource Languages like Hindi.
+"""
+
+_HOMEPAGE = "https://github.com/midas-research/hindi-nli-data"
+
+_LICENSE = """
+MIT License
+
+Copyright (c) 2019 MIDAS, IIIT Delhi
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+
+_TRAIN_DOWNLOAD_URL = (
+    "https://raw.githubusercontent.com/avinsit123/hindi-nli-data/master/Textual_Entailment/HDA/HDA_recasted_train.tsv"
+)
+_VALID_DOWNLOAD_URL = (
+    "https://raw.githubusercontent.com/avinsit123/hindi-nli-data/master/Textual_Entailment/HDA/HDA_recasted_dev.tsv"
+)
+_TEST_DOWNLOAD_URL = (
+    "https://raw.githubusercontent.com/avinsit123/hindi-nli-data/master/Textual_Entailment/HDA/HDA_recasted_test.tsv"
+)
+
+
+class HdaNliHindiConfig(datasets.BuilderConfig):
+    """BuilderConfig for HDA NLI Hindi Config"""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for HDA NLI Hindi Config
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(HdaNliHindiConfig, self).__init__(**kwargs)
+
+
+class HdaNliHindi(datasets.GeneratorBasedBuilder):
+    """HDA NLI Hindi dataset -- Dataset providing textual-entailment pairs for NLI tasks in Hindi"""
+
+    BUILDER_CONFIGS = [
+        HdaNliHindiConfig(
+            name="hda nli hindi",
```

Comment:
```suggestion
            name="HDA nli hindi",
```
Did you rename the config at one point ?
In the datasets_infos.json je config_name is `HDA nli hindi`, not `hda nli hindi`